x-pack/package.json

```diff
@@ -42,6 +42,7 @@
     "@types/graphql": "^0.13.1",
     "@types/hapi": "^17.0.18",
     "@types/history": "^4.6.2",
+    "@types/inline-style-prefixer": "3.0.1",
```

Comment:
If the build isn't cranky about it and it works with this in just our plugin that would be better until another plugin wants to use it. No changes required.